libs/expo/betterangels/src/lib/hooks/californiaIdUniqueCheck/ClientCaliforniaUniqueCheck.graphql

```diff
@@ -0,0 +1,7 @@
+query ClientCaliforniaUniqueCheck($filters: ClientProfileFilter) {
```

Comment:
Hi @mikefeldberg

I didn't get to see the visual feedback after I submitted a CA ID. I tried again by rescanning this QR code to refresh and start over again but I get the same result.
The visual feedback: what I meant is after tapping the Submit button, I expected the page guides me to view-only mode of Client Profile page but it didn't happen.

Although as I tapped the Back button (thinking not saving the ID info), the info I entered was there. So I know it's saved but somehow, it just doesn't behave the way it should be.

It's happening on both, Android and iOS.

Please check. thanks!

Reply:
Thanks, @mikefeldberg 

I confirmed the submit action takes me to the view-only mode page now. 

One more thing I noticed though.

This is only happening on iOS (Android was fine). 

When I submitted the invalid CA ID, I see the validation message at the beginning. But if I ignored it and tapped on the Submit button anyway, the message disappears. But the error snicker bar (alert) shows up (understandable). My questions is: shouldn't the validation message be persistent till the user enters the valid format or remove the entry?

Thanks!

Reply:
@shiozakiyuko I see what happened here. fixed!